src/robusta/core/playbooks/internal/ai_integration.py

```diff
@@ -320,49 +391,59 @@ def holmes_issue_chat(event: ExecutionBaseEvent, params: HolmesIssueChatParams):
             failure=False,
         )
         finding.add_enrichment(
-            [HolmesChatResultsBlock(holmes_result=holmes_result)], enrichment_type=EnrichmentType.ai_analysis
+            [HolmesChatResultsBlock(holmes_result=holmes_result)],
+            enrichment_type=EnrichmentType.ai_analysis,
         )
 
         event.add_finding(finding)
 
     except Exception as e:
-        logging.exception(f"Failed to get holmes chat for {conversation_title}", exc_info=True)
+        logging.exception(
+            f"Failed to get holmes chat for {conversation_title}", exc_info=True
+        )
         handle_holmes_error(e)
 
 
 @action
 def holmes_chat(event: ExecutionBaseEvent, params: HolmesChatParams):
     holmes_url = HolmesDiscovery.find_holmes_url(params.holmes_url)
     if not holmes_url:
-        raise ActionException(ErrorCodes.HOLMES_DISCOVERY_FAILED, "Robusta couldn't connect to the Holmes client.")
+        raise ActionException(
+            ErrorCodes.HOLMES_DISCOVERY_FAILED,
+            "Robusta couldn't connect to the Holmes client.",
+        )
 
     cluster_name = event.get_context().cluster_name
 
     try:
         holmes_req = HolmesChatRequest(
-            ask=params.ask, 
-            conversation_history=params.conversation_history, 
-            model=params.model, 
-            stream=params.stream, 
+            ask=params.ask,
+            conversation_history=params.conversation_history,
+            model=params.model,
+            stream=params.stream,
             additional_system_prompt=params.additional_system_prompt,
             enable_tool_approval=params.enable_tool_approval,
             tool_decisions=params.tool_decisions,
         )
         url = f"{holmes_url}/api/chat"
         if params.stream:
-            with requests.post(
-                url,
-                data=holmes_req.json(),
-                stream=True,
-                headers={"Connection": "keep-alive"},
-            ) as resp:
-                resp.raise_for_status()
-                for line in resp.iter_content(
-                    chunk_size=None, decode_unicode=True
-                ):  # Avoid streaming chunks from holmes. send them as they arrive.
-                    if line:
-                        event.ws(data=line)
-            return
+            if params.render_graph_images:
```

Comment:
I think we want to do most of the same logic in both flows except for the last step
i would pass render_graphs to stream_and_render_graphs and 
change the 
`if not tool_res or tool_res.get("name", "") not in
`
to
 
`if not stream_and_render_graphs or not tool_res or tool_res.get("name", "") not in...
`
so that way we dont need two stream flows, wdyt?